apps/frontend/src/index.tsx

```diff
@@ -38,6 +38,14 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 								element: <Auth />,
 								path: AppRoute.SIGN_UP,
 							},
+							{
+								element: <Auth />,
+								path: AppRoute.FORGOT_PASSWORD,
+							},
+							{
+								element: <Auth />,
+								path: `${AppRoute.RESET_PASSWORD}/:token`,
```

Comment:
```suggestion
								path: `${AppRoute.RESET_PASSWORD}`,
```
Send token as query params (`?token=xxx`)